ktlint/src/main/kotlin/com/pinterest/ktlint/Main.kt

```diff
@@ -262,9 +263,7 @@ class KtlintCommandLine {
         val start = System.currentTimeMillis()
         // load 3rd party ruleset(s) (if any)
         val dependencyResolver = lazy(LazyThreadSafetyMode.NONE) { buildDependencyResolver() }
-        if (!rulesets.isEmpty()) {
-            loadJARs(dependencyResolver, rulesets)
-        }
+        val resultSetClassLoader = getServiceClassloader(dependencyResolver, rulesets)
```

Comment:
Did you mean `rulesetClassloader`?

Reply:
It was originally like that in [your commit](https://github.com/pinterest/ktlint/pull/351/commits/6651111d7d6395848ece67fa72a418c4cc0de8d8#diff-fb157ca11660a4b329fbc3faf6c2ed5eR275), but yeah, `rulesetClassLoader` makes more sense!